java/sage/plugin/CorePluginManager.java

```diff
@@ -2182,14 +2066,7 @@ else if ("FanArt".equals(packages[i].type))
     }
     finally
     {
-      if (stagedRenameWriter != null)
-      {
-        try
-        {
-          stagedRenameWriter.close();
-        }
-        catch (Exception e){}
-      }
+      IOUtils.closeQuietly(stagedRenameWriter);
```

Comment:
Basically, stagedRenameWriter was a writer before, and now it holds a reference to the writer.   the wrapper implements closeable so closeQUietly is doing the same as the lines it is replacing.    The stagedRenameWriter.close() method closes the underlying implementation.  Does it not??